src/clr-core/internal/base/button.base.ts

```diff
@@ -74,8 +84,8 @@ export class CdsBaseButton extends LitElement {
    */
   private setupNativeButtonBehavior() {
     this.appendHiddenButton();
-    this.addEventListener('click', e => this.triggerNativeButtonBehavior(e));
-    this.addEventListener('keydown', e => this.emulateKeyBoardEventBehavior(e));
+    this.addEventListener('click', this.triggerNativeButtonBehavior);
+    this.addEventListener('keydown', this.emulateKeyBoardEventBehavior);
```

Comment:
Add corresponding `removeEventListener`s in disconnectedCallback?

Reply:
That's only necessary if we are setting an event listener on something that is not the host element or "inside" the host element. Like if we set up an event listener on document.

All browsers we support clean up after HTMLElements after they have been destroyed/removed.